src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
About categorization: if I define ``E_CUSTOM`` const and I want get E_* predefined constants with ``get_defined_constants()['Core']``, my ``E_CUSTOM`` (which is in ``['user']``) will be not dumped.